OpenLIFULib/OpenLIFULib/algorithm_input_widget.py

```diff
@@ -103,7 +103,9 @@ def _populate_from_loaded_objects(self) -> None:
         else:
             self.volume_input.combo_box.setEnabled(True)
             for volume_node in slicer.util.getNodesByClass('vtkMRMLScalarVolumeNode'):
-                self.add_volume_to_combobox(volume_node)
+                # Check that the volume is not an OpenLIFUSolution output volume
+                if not volume_node.GetAttribute('isOpenLIFUSolution'):
```

Comment:
```suggestion
                if volume_node.GetAttribute('isOpenLIFUSolution') is None:
```

Just for clarity about what's happening, since it's really `None` getting converted to `False` and `'True'` getting converted to `True`